__tests__/auth.spec.js

```diff
@@ -1,13 +1,13 @@
 require('dotenv').config()
 const request = require('supertest');
-const server = require("../server/app");
 const { expect } = require('chai');
-const seed = require('./seed');
 const log = require('loglevel');
-const Transfer = require("../server/models/Transfer");
-const TrustRelationship = require("../server/models/TrustRelationship");
 const sinon = require("sinon");
 const chai = require("chai");
+const server = require("../server/app");
+const seed = require('./seed');
+const Transfer = require("../server/models/Transfer");
+const TrustRelationship = require("../server/models/TrustRelationship");
```

Comment:
So the linter will automatically sort the imports?

Reply:
if set up correctly then yes. I've imported a library that can sort some automatically, but not all of them, I investigated a bit, but then decided to create a separate issue and tackle it there